src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ExceptionHandlerTests.kt

```diff
@@ -58,6 +58,78 @@ class ExceptionHandlerTests : SecureIntegrationTests()
         Assertions.assertThat(entity.body).contains("404 Error Page")
     }
 
+    @Test
+    fun `adr resource error are correctly formatted in English`()
+    {
+        expectTranslatedAdrException(
+            "adrResourceError",
+            "Unable to load resource, check resource in ADR ${errorArgs.first()}.",
+            Language.EN,
+        )
+    }
+    @Test
+    fun `adr resource error are correctly formatted without adrURl`()
+    {
+        expectTranslatedAdrException(
+            "adrResourceError",
+            "Unable to load resource, check resource in ADR .",
+            Language.EN,
+            ""
+        )
+    }
+
+    @Test
+    fun `adr resource error are correctly formatted in French`()
+    {
+        expectTranslatedAdrException(
+            "adrResourceError",
+            "Impossible de charger la ressource, vérifiez la ressource dans ADR ${errorArgs.first()}.",
+            Language.FR
+        )
+    }
+
+    @Test
+    fun `adr resource error are correctly formatted in Portuguese`()
+    {
+        expectTranslatedAdrException(
+            "adrResourceError",
+            "Não é possível carregar o recurso, verifique o recurso no ADR ${errorArgs.first()}.",
+            Language.PT
+        )
+    }
+
+    @Test
+    fun `adr no permission to load resource error are correctly formatted in English`()
+    {
+        expectTranslatedAdrException(
+            "noPermissionToAccessResource",
+            "You do not have permission to load this resource from ADR. Contact dataset admin for permission.",
+            Language.EN,
+        )
+    }
+
+    @Test
+    fun `adr no permission to load resource error are correctly formatted in French`()
+    {
+        expectTranslatedAdrException(
+            "noPermissionToAccessResource",
+            "Vous nêtes pas autorisé à charger cette ressource à partir dADR. " +
+                    "Contacter ladministrateur de lensemble de données pour obtenir lautorisation",
```

Comment:
This looks like it has lost `'` punctuation? .e.g `d'ADR`, `l'administrateur`. `l'ensemble' etc. I think for multiline strings in the ErrorMessageBundle.properties these need to be escaped with another `'` e.g. `l''ensemble'`. `unexpectedError` has an example

Reply:
Nice spot. I was unsure why I wasn't getting the punctuations in the test.